src/utils/daos.c

```diff
@@ -927,6 +1014,23 @@ call_dfuse_ioctl(char *path, struct dfuse_il_reply *reply)
 	return 0;
 }
 
+static int
+fs_op_hdlr(struct cmd_args_s *ap)
```

Comment:
We need to make sure both src and dst are provided.
For example:
`daos fs copy` gives "dfs_lookup / failed"
But really, we should make sure we have the required arguments first.

Reply:
@daltonbohning I think we can save this for the next iteration. I will add a TODO since this I think deserves its own function to check all the arguments.

Reply:
@dsikich Sure! There will also be a test case for this as well